README.md

```diff
@@ -391,12 +394,12 @@ url_for('/css/style.css', {relative: false})
 
 ## bind(hexo)
 
-Following utilities require `bind(hexo)` / `bind(this)` to parse the user config when initializing:
+Following utilities require `bind(hexo)` / `bind(this)` / `call(this, input)` to parse the user config when initializing:
```

Comment:
```suggestion
Following utilities require `bind(hexo)` / `bind(this)` / `call(hexo, input)` / `call(this, input)` to parse the user config when initializing:
```

Since there is a `bind(hexo)`, the `call(hexo, input)` might need to be added as well.